OpenData/secular printed material - 1921 - 1990/Coraa ny Gael/Coraa ny Gael 1953-08-25/document.csv

```diff
@@ -1,31 +1,31 @@
 ﻿Manx,English,Notes
 CORAA NY GAEL,The Voice of the Gaels,
      Ta’n Briw Sir Percy Cowley as e ven er chee goaill turrys trooid yn Mediterranean.,     The judge Sir Percy Cowley and his wife are about to take a tour through the Mediterranean.,
-     Nee Yn Cheshaght Fendeilys ny Cheerey Vanninagh goaill turrys fud ny hellan Jerdein shoh cheet. Mastey ny buill ver ad shilley er ta Tholt y Will as Glion Balleyglass.,     The Manx Civil Defence Force will take tour throughout the island this coming Thursday. Amongst the places they will see are Thotly-Will and Ballaglass Glen.,
-"      T’ad gra dooys dy vel yn Cohionnal dy Vanninee ayns America Twoaieagh ec Cleveland, er ve speeideilys mooar. ","     They say to me that the Conference of Manx people in North America at Cleveland, has been a great success. ",
+     Nee Yn Cheshaght Fendeilys ny Cheerey Vanninagh goaill turrys fud ny hellan Jerdein shoh cheet. Mastey ny buill ver ad shilley er ta Tholt y Will as Glion Balleyglass.,     The Manx Civil Defence Force will take a tour throughout the island this coming Thursday. Amongst the places they will see are Tholty-Will and Ballaglass Glen.,
```

Comment:
This all looks great, thanks! One tiny nit: I believe the place is called Tholt-y-Will with two hyphens.